src/index.js

```diff
@@ -449,7 +416,7 @@ async function run (input, output, { puppeteerConfig = {}, quiet = false, output
     if (input && /\.(md|markdown)$/.test(input)) {
       const imagePromises = []
       for (const mermaidCodeblockMatch of definition.matchAll(mermaidChartsInMarkdownRegexGlobal)) {
-        const mermaidDefinition = mermaidCodeblockMatch[1]
+        const mermaidDefinition = mermaidCodeblockMatch[2]
```

Comment:
The bug here was the regex has two match groups, where **Group 1** contains a leading newline! **Group 2** is the correct group to use, since it doesn't contain the `\n` character.

Normally, when using HTML, you can put as many newlines as you want and the browser will remove everything (so this bug didn't matter), but the `mermaid.render()` API is much stricter.

See 
![image](https://user-images.githubusercontent.com/19716675/236344598-38e12fbb-3ab8-49e4-a53a-50acabf9110e.png)